### PR TITLE
Change color and icon of SD category + fix hardcoded accent colors

### PR DESCRIPTION
**Pull request by joeyballentine**

### CODE REVIEW

File: src/renderer/components/NodeSelectorPanel/RepresentativeNode.tsx [hunk 1]
```diff
@@ -26,8 +27,11 @@ export const RepresentativeNode = memo(
         collapsed = false,
         createNodeFromSelector,
     }: RepresentativeNodeProps) => {
+        const { categories } = useContext(BackendContext);
+
         const bgColor = 'var(--selector-node-bg)';
-        const accentColor = getNodeAccentColor(category);
+        const currentCategory = categories.find((c) => c.name === category);
+        const accentColor = currentCategory?.color ?? '#CCCCCC';
```
Comment (RunDevelopment) [inline src/renderer/components/NodeSelectorPanel/RepresentativeNode.tsx:34]: The fallback color will be pretty much invisible in light mode.

**Review by RunDevelopment — COMMENTED** [review]
Instead of duplicating `categories.find((c) => c.name === category)?.color ?? '#CCCCCC'` everywhere, could you extract it into a function `getNodeAccentColor(category: string | undefined) -> colorString` in `BackendContext`?

File: src/renderer/helpers/accentColors.ts [hunk 2]
```diff
@@ -39,3 +40,7 @@ export const getTypeAccentColors = (inputType: Type): readonly [string, ...strin
     }
     return colors.length > 0 ? (colors as [string, ...string[]]) : defaultColorList;
 };
+
+export const getCategoryAccentColor = (categories: Category[], category: string) => {
```
Comment (RunDevelopment) [inline src/renderer/helpers/accentColors.ts:44]: We don't mutate the array.

```suggestion
export const getCategoryAccentColor = (categories: readonly Category[], category: string) => {
```

File: src/renderer/hooks/usePaneNodeSearchMenu.tsx [hunk 3]
```diff
@@ -48,16 +48,16 @@ import { useThemeColor } from './useThemeColor';
 interface SchemaItemProps {
     schema: NodeSchema;
     isFavorite?: boolean;
+    accentColor?: string;
```
Comment (RunDevelopment) [inline src/renderer/hooks/usePaneNodeSearchMenu.tsx:51]: Why is this optional? We always pass in an accent color.

**Comment by joeyballentine** [comment]
I'm just going to assume this is good now

**Pull request merged**